docs/chainlink-nodes/configuration-variables.md

```diff
@@ -1560,6 +1562,12 @@ Example: `P2P_PEER_ID=12D3KooWMHMRLQkgPbFSYHwD3NBuwtS1AmxhvKVUrcfyaGDASR4U`
 
 ### Networking Stack V2
 
+> 🚧 If using the Networking Stack V2, you must unset the following Networking Stack V1 configuration variables:
+>
+> - [P2P_LISTEN_PORT](#p2p_listen_port)
```

Comment:
@jmank88 should we list all of the other networking stack v1 env vars here too?

Reply:
Added

Reply:
Fixed